flake.nix

```diff
@@ -37,9 +37,10 @@
                     ps
 
                     # typescript
+                    python3
                     nodePackages.pnpm
-                    nodePackages.yarn
-                    nodejs-18_x
+                    nodejs_20
+                    (yarn.override { nodejs = nodejs_20; })
```

Comment:
Yarn comes with node 18 so we need to override it here. Lodestar 1.12.0 doesn't build with 18.16.1

Reply:
Btw the lodestar in our repo is the forked branch https://github.com/Snowfork/lodestar/tree/snowbridge which is based on 1.14.0 with a hack to work with node 18.16.

Anyway, the hack here is necessary for working with node 20 so happy to see it.